tests/unit-tests/dds/DCPS/ServiceEventDispatcher.cpp

```diff
@@ -273,7 +273,7 @@ TEST(dds_DCPS_ServiceEventDispatcher, TestShutdown)
   test_event->wait(1u);
   dispatcher->shutdown();
 
-  EXPECT_EQ(false, dispatcher->dispatch(test_event));
+  EXPECT_TRUE(0 == dispatcher->dispatch(test_event));
```

Comment:
Confused by this change. The API for `EventDispatcher::dispatch()` returns a `bool` to indicate success, and it should return 'false' after `shutdown` has been called on the dispatcher. What was the warning message here?

Reply:
I saw it in the Coverity logs before.  I think there's something with how the GTest macros work.
I suggest trying `EXPECT_FALSE(dispatcher->dispatch(test_event))` since it returns bool